includes/rest/class-users.php

```diff
@@ -40,6 +42,25 @@ public static function register_routes() {
 				),
 			)
 		);
+
+		\register_rest_route(
+			ACTIVITYPUB_REST_NAMESPACE,
+			'/users/(?P<user_id>[\w\-\.]+)/remote-follow',
```

Comment:
Do we really need a `user_id` in this URL? What is it doing for us?

(looks at the code below)

Oh. :) That. Carry on!

Reply:
we need the ID, because we need the identifier of that user to be passed as param for the remote-follow endpoint.

Example: `https://mastodon.social/authorize_interaction?uri=username.wordpress.com@username.wordpress.com`